orchestration/templates/ingest-dog-aging.yaml

```
@@ -0,0 +1,97 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ingest-dog-aging
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: gcs-prefix
+          {{- $gcsPrefix := "{{inputs.parameters.gcs-prefix}}" }}
+      dag:
+        tasks:
+          - name: run-extraction
+            template: run-extraction-pipeline
+            arguments:
+              parameters:
+                - name: api-token
+                  value: {{ .Values.redcap.apiKey }}
+                - name: output-prefix
+                  value: '{{ printf "%s/raw" $gcsPrefix }}'
+          - name: run-transformation
+            dependencies: [run-extraction]
+            template: run-transformation-pipeline
+            arguments:
+              parameters:
+                - name: input-prefix
+                  value: '{{ printf "%s/raw" $gcsPrefix }}'
+                - name: output-prefix
+                  value: '{{ printf "%s/processed" $gcsPrefix }}'
+
+
+    {{- /* Version used for both Dataflow docker images. */}}
+    {{- $version := default "latest" .Chart.AppVersion }}
+    {{- $bucket := .Values.gcs.bucketName }}
+    {{- with .Values.dataflow }}
+    ##
+    ## Template used to launch a Dataflow processing job that pulls
+    ## all relevant metadata from REDCap.
+    ##
+    - name: run-extraction-pipeline
+      inputs:
+        parameters:
+          - name: api-token
+          {{- $apiToken := "{{inputs.parameters.api-token}}" }}
+          - name: output-prefix
+          {{- $outputPrefix := "{{inputs.parameters.output-prefix}}" }}
+      container:
+        image: us.gcr.io/broad-dsp-gcr-public/dog-aging-extraction-pipeline:{{ $version }}
+        command: []
+        args:
+          - --runner=dataflow
+          - --outputDir=gs://{{ $bucket }}/{{ $outputPrefix }}
+          - --apiToken={{ $apiToken }}
```

This should reference the injected environment variable. I think the kube syntax is:
```suggestion
          - --apiToken=$(API_TOKEN)
```